app/templates/components/duke-ds-user-search-fields.hbs

```diff
@@ -0,0 +1,13 @@
+{{#bs-form formLayout='vertical' model=this onSubmit=(action 'search') as |form|}}
+  {{#form.element controlType="text" label=labelText property="query" placeholder=kind.placeholder  as |el|}}
+    <div class="input-group">
+      <div class="input-group-addon">Search By&nbsp;
+        {{#each kinds as |k|}}
+          <label class="input-group-text mode">{{radio-button value=k checked=kind class=k.mode}} <span>{{k.name}}</span></label>
```

Comment:
What does the `mode` class do here?

Reply:
It's just used to make the `label`s easy to find in tests: 

https://github.com/Duke-GCB/datadelivery-ui/blob/371e5f59e1606b71f9c2f85d79bdeb62768132b0/tests/integration/components/duke-ds-user-search-fields-test.js#L11